#TPointMove

#Original position
float32 Pox
float32 Poy

#Original size
float32 So

#Displacement of position
float32 DPx
float32 DPy

#Displacement of size
float32 DS
